pages/_app.tsx

```diff
@@ -1,6 +1,10 @@
 import type { AppProps } from "next/app"
 
+<<<<<<< HEAD
 import "@/firebase"
+=======
+import "../firebase"
+>>>>>>> ad94ee42c4dc1f008d323396c09fb38fd71e81ee
```

Comment:
que raro esto... en dado caso quedaria `import "@/firebase"`

Reply:
Esto es un merge conflict, seguramente lo commiteaste sin querer. Elegí uno de los dos, y borrá el otro.

https://css-tricks.com/merge-conflicts-what-they-are-and-how-to-deal-with-them/